gen/drupal_elixir

```diff
@@ -67,9 +67,9 @@ my $fileMemberships = $DIRECTORY . "memberships.csv";
 my $agent = perunServicesInit->getAgent;
 my $groupAgent = $agent->getGroupsAgent;
 
-foreach my $resourceData ($data->getChildElements) {
-	foreach my $groupData (($resourceData->getChildElements)[0]->getChildElements) {
-		my $groupMembersLogins = processGroups $groupData;
+foreach my $resourceId ( $data->getResourceIds() ) {
+	foreach my $groupId ( $data->getGroupIdsForResource( resource => $resourceId ) ) {
+		my $groupMembersLogins = processGroups $groupId, $resourceId;
```

Comment:
Variable `$groupMembersLogins` is not used, you can simply call `processGroups()` method without processing returned value.

Reply:
removed variable